webpack/build.config.js

```diff
@@ -0,0 +1,31 @@
+var srcFolder = './src';
+var distFolder = './dist';
+var pluginsFolder = './plugins';
+var externalPluginsTmpFolder = './.external_plugins';
+
+var dirs = {
+  src: srcFolder,
+  dist: distFolder,
+  plugins: pluginsFolder,
+  pluginsTmp: externalPluginsTmpFolder,
+  srcJS: srcFolder + '/js',
+  distJS: distFolder,
+  srcCSS: srcFolder + '/styles',
+  distCSS: distFolder,
+  srcImg: srcFolder + '/img',
+  distImg: distFolder + '/img'
+};
+
+var files = {
+  srcJS: dirs.srcJS + '/index.js',
+  distJS: dirs.distJS + '/index.js',
+  srcCSS: dirs.srcCSS + '/index.less',
+  distCSS: dirs.distCSS + '/index.css',
+  srcHTML: dirs.src + '/index.html',
+  distHTML: dirs.dist + '/index.html'
+};
```

Comment:
You're right, I'd say we get rid of this file because it is now an unnecessary layer of abstraction.